src/app/main.cpp

```diff
@@ -118,6 +118,12 @@ int main(int argc, char *argv[])
         // Create Application
         auto app = std::make_unique<Application>(argc, argv);
 
+#ifdef Q_OS_WIN
+        // QCoreApplication::applicationDirPath() needs an Application object instantiated first
+        // Let's hope that there won't be a crash before this line
+        qputenv("_NT_ALT_SYMBOL_PATH", Application::applicationDirPath().toLocal8Bit());
```

Comment:
After rereading the documentation, I realized that use of `_NT_SYMBOL_PATH` is preferred, and use of `_NT_ALT_SYMBOL_PATH` is recommended for purely specific cases.
In addition, since this is not a purely native qBittorrent variable, it would be nice not to overwrite it (if it is already set outside of qBittorrent), but to append qBittorrent path to the existing value.

Reply:
Done.